pkg/scalers/scaler.go

```diff
@@ -105,3 +109,33 @@ func GetFromAuthOrMeta(config *ScalerConfig, field string) (string, error) {
 func GenerateMetricNameWithIndex(scalerIndex int, metricName string) string {
 	return fmt.Sprintf("s%d-%s", scalerIndex, metricName)
 }
+
+// GetExternalMetricTargetType helps getting the metric target type of the scaler
+func GetExternalMetricTargetType(config *ScalerConfig) (v2beta2.MetricTargetType, error) {
```

Comment:
Again, I'd probably omit `External` in here, but it is just a nit. If you prefer to keep it there, I'd fine wit it...probably 😄